### ISSUE TITLE
Upload of incoming drawings 

### ISSUE THREAD

**Issue by StephenHiggon01** [nittygritty-uk/Atvero-Issues#242]
![upload failure](https://user-images.githubusercontent.com/44202559/52282184-b6f65000-2957-11e9-8de5-f6a9803a9a66.PNG)

Millbrook Park Project - Uploading a incoming drawings onto this project (example number MBPMS-AA-00-00-DR-A-0200) - system will not allow upload.

**Comment by allanchaplin** [comment]
If the parsing of a received file fails to match the place holder, accept it with no fields populated.

Fixed in PR#264

**Comment by StephenHiggon01** [comment]
The main search filter will be 'Document Type' so it is a key requirement to tag what document type is being checked into the system. Will that work with the 'no fields populated' solution?

**Comment by allanchaplin** [comment]
Hi Stephen,

I've re-opened this issue as we have a question.

This change effectively allows setting metadata that does not match the filename?

Is this acceptable?

Thanks,

Allan

**Comment by StephenHiggon01** [comment]
Yes. It is key to define what document type an incoming document is otherwise it will not be searchable by filter. The document types are effectively replacing the document pools in our case. My view is it is a key field to define at upload. 

 An example - If a document is sent '0124-O&M-22' it is still a certain type document and it is the user that should define what document type upon upload.
